src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -33,21 +33,219 @@
+    }
+
+    /**
+     * @description Count the number of DatedConversionRate records with an effective date in the last 180 days.
+     * If greater than 0, then consider AdvancedMultiCurrency to be enabled within an organization. This accounts for
+     * the fact that this object is always visible, but may not actually have any valid/recent data in it; and because
+     * there is no plan for core to ever fix this bug.
+     */
+    private void handleIsAdvancedMulticurrencyEnabled() {
+        Boolean isEnabled=false;
+        if (UserInfo.isMultiCurrencyOrganization() == true) {
+            try {
+                Integer DCRCount = Database.query('SELECT Id FROM DatedConversionRate WHERE StartDate = LAST_N_DAYS:180 LIMIT 1').size();
+                if (DCRCount == 1) {
+                    isEnabled = true;
+                }
+            } catch (Exception e) { }
+        }
+        System.FeatureManagement.setPackageBooleanValue('Org_IsEnabled_AdvancedMultiCurrency', isEnabled);
+    }
+
+    /**
+     * @description Count the number of non-english languages associated with active standard users.
+     */
+    private void handleNonEnglishLanguageCount() {
+        Set<String> languages = new Set<String>();
+        for (User u : [SELECT LanguageLocaleKey
+                FROM User
+                WHERE IsActive = TRUE
+                AND UserType = 'Standard'
+                AND LanguageLocaleKey != 'en_US'
+                ORDER BY LanguageLocaleKey]
+        ) {
+            if (u.LanguageLocaleKey != null && !u.LanguageLocaleKey.startsWithIgnoreCase('en_')) {
+                languages.add(u.LanguageLocaleKey);
+            }
+        }
+        System.FeatureManagement.setPackageIntegerValue('Org_Count_of_NonEnglishLanguages', languages.size());
+    }
+
+    /**
+     * @description Are there are any TDTM Trigger Handler records marked as Inactive or that the custom has added.
+     */
+    private void handleHasUserManagedTriggers() {
+        Boolean hasUserManagedTrigger = false;
+        for (Trigger_Handler__c th : TDTM_Config_API.getCachedRecords()) {
+            if (th.User_Managed__c == true || th.Active__c == false) {
+                hasUserManagedTrigger = true;
+                break;
+            }
+        }
+        System.FeatureManagement.setPackageBooleanValue('getHasUserManagedTriggers', hasUserManagedTrigger);
+    }
+
+    /**
+     * @description Total number of records in the Error__c custom object
+     */
+    private void handleErrorLogCount() {
+        try {
+            Integer count = [SELECT Count() FROM Error__c];
+            System.FeatureManagement.setPackageIntegerValue('Data_Count_of_ErrorLog', count);
+        } catch (Exception ex) {}
+    }
+
+    /**
+     * @description Total number of Opportunities associated with a Recurring Donation
+     */
+    private void handleCountOfRecurringDonationOpps() {
+        try {
+            Integer count = [SELECT Count() FROM Opportunity
```

Comment:
Is it possible to re-write SOQLs in this class by the latest coding style?

Reply:
Yes, and thanks for holding me to that. It's easy to forget that we have some preferred styles, though at the same time I don't really think there's anything wrong with having the FROM on the same line as the SELECT if there's only a single field.